interpreter/Makefile

```diff
@@ -88,6 +88,13 @@ install:
 	dune build -p $(NAME) @install
 	dune install
 
+.PHONY: opam-release
```

Comment:
I don't think this works, since the target contains a wildcard.

Reply:
Oh yes sorry, it's because at first I had a `opam-release:` target without a Makefile.